packages/framework/src/Model/Product/Elasticsearch/ExportCronModule.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Model\Product\Elasticsearch;
+
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Component\Elasticsearch\AbstractExportCronModule;
+use Shopsys\FrameworkBundle\Component\Elasticsearch\IndexDefinitionLoader;
+use Shopsys\FrameworkBundle\Component\Elasticsearch\IndexFacade;
+
+class ExportCronModule extends AbstractExportCronModule
```

Comment:
class name `ExportCronModule` is too general in my point of view. It is like rename `ProductFacade` to `Facade`... Namespace for specification is too small...

Reply:
Renamed to `ProductExportCronModule` (and other similar classes were renamed too)